src/components/DynamicForm/components/DynamicFormWrapper.jsx

```diff
@@ -102,6 +104,7 @@ DynamicFormWrapper.propTypes = {
 };
```

Comment:
Not needed atm, but consider adding options here as well as to DynamicForm.proptypes.
`client.mutate({ mutation, variables, ...options})`

Reply:
what would options be? ill look into adding it